### DISCUSSION TITLE
关于国足的事

### DISCUSSION THREAD

**Discussion by Enigma-Soul** [Meloong-Git/PCL#1146] [提问] [answered]
联机什么时候好啊，会不会联机做好了国足就已经踢进世界杯了啊
到底有多难哪，3个人两个月了还没好....

**Comment by Pigeon0v0** [comment, marked as answer]
> 联机什么时候好啊，会不会联机做好了国足就已经踢进世界杯了啊
> 
> 到底有多难哪，3个人两个月了还没好....

有一些许可证相关的事情，这个是耗时较长的

**Reply by Enigma-Soul** [reply]
是不是经费不足，买不起服务器🤣🤣🤣
